.github/workflows/test.yml

```diff
@@ -27,7 +31,7 @@ jobs:
       - name: Run unit tests with code coverage and Thread Sanitizer
         run: swift test --enable-test-discovery --filter=^PostgresNIOTests --sanitize=thread --enable-code-coverage
       - name: Submit coverage report to Codecov.io
-        if: "!contains(matrix.container, 'nightly')"
+        if: ${{ !contains(matrix.container, '5.8') }}
```

Comment:
why did we change this to `5.8`?

Reply:
The reason we used 'nightly' was that Codecov seemed to have problems with nightly builds. 
We can either try to see if Codecov now works fine with nightly builds, or let it stay disabled to make sure it doesn't fail the CI. 
I think Gwynne thought now that 5.8 is released, we can change that to '5.8', but if i were to guess i would say the problems we were having were nightly related, not 5.8's fault.

Reply:
I see Gwynne is tracking the Codecov issue already, based on her comment above. Then i guess the options are to either remove that 'if' or change it back.

Reply:
The issue was that 5.8 changed the behavior of the `--show-codecov-path` flag. This was fixed, but the fix didn't make it in until 5.9. See https://github.com/apple/swift-package-manager/issues/5853 for details.